query_panel/main.js

```diff
@@ -24,8 +24,33 @@ class Grid {
     this.elem = document.querySelector("perspective-viewer");
   }
 
-  async load(rows) {
-    const table = await this.worker.table(rows);
+  mapType(agateType) {
+    switch (agateType) {
+      case "Text":
+        return "string";
+      case "Integer":
+        return "integer";
+      case "Boolean":
+        return "boolean";
+      case "Date":
+        return "date";
+      case "DateTime":
+        return "datetime";
```

Comment:
for these can we map to strings?

Reply:
I think it is not necessary. I think datetime is OK to show in user timezone. date is not shown in user timezone.